CODE_OF_CONDUCT.md

```diff
@@ -12,9 +12,9 @@ orientation.
 ## Our Standards
 
 Examples of behaviour that contributes to creating a positive environment
-include:
+including:
 
-*   Using welcoming and inclusive language.
+*   Using welcoming and impressive language.
```

Comment:
We want the to language to be `inclusive` in the sense we are welcoming people from different backgrounds and varying technical levels to contribute to the project.
Please revert it to `inclusive`